sql/mdl.h

```diff
@@ -366,16 +364,7 @@ class MDL_key
   /**
     Compare two MDL keys lexicographically.
   */
-  int cmp(const MDL_key *rhs) const
-  {
-    /*
-      The key buffer is always '\0'-terminated. Since key
-      character set is utf-8, we can safely assume that no
-      character starts with a zero byte.
-    */
-    using std::min;
-    return memcmp(m_ptr, rhs->m_ptr, min(m_length, rhs->m_length));
```

Comment:
Just use `MY_MIN()` and be done with it..

Reply:
I would be glad to use more C++ headers in code but actually do the opposite :)

Updated.